addons/danger/functions/fnc_isAlive.sqf

```diff
@@ -0,0 +1,19 @@
+#include "script_component.hpp"
+/*
+ * Author: nkenny
+ * Checks if a Unit is Alive and Awake
+ *
+ * Arguments:
+ * 0: Unit being checked <OBJECT>
+ *
+ * Return Value:
+ * boolean
+ *
+ * Example:
+ * bob call lambs_danger_fnc_isAlive;
+ *
+ * Public: No
+*/
+alive _this
+|| {!(_this getVariable ["ACE_isUnconscious", false])}
```

Comment:
This one is not needed anymore, unless you want to support ACE versions before the Medical rewrite, or people running old ace medical.

Reply:
I think we should support it because there are still a few communities running ace 3.12.x